bin/pyrdp-mitm.py

```diff
@@ -50,10 +50,11 @@ def main():
     parser.add_argument("--crawler-ignore-file", help="File to be used by the crawler to chose what folders to avoid when scraping the client shared drives.", default=None)
     parser.add_argument("--no-replay", help="Disable replay recording", action="store_true")
     parser.add_argument("--no-downgrade", help="Disables downgrading of unsupported extensions. This makes PyRDP harder to fingerprint but might impact the player's ability to replay captured traffic.", action="store_true")
+    parser.add_argument("--test", help="Test the initialization of the program. Initialize the MITM, then exit without listening for connections.", action="store_true")
```

Comment:
I don't like the idea of having a switch just for the purpose of testing... wouldn't it be better to have a test harness that initializes the core components without starting the reactor and exiting cleanly? This would probably require some refactoring, but would be much cleaner than test-specific code paths and logic.